tests/4.5/offloading_success.F90

```diff
@@ -20,6 +20,6 @@ PROGRAM offloading_success
     print*, "Target region executed on the device"
   END IF
 
-  CALL EXIT(errors)
+  if( errors .ne. 0 ) stop 1
```

Comment:
why not use error stop here ?

Reply:
Error stop should be fine, too. I can change to that if you want. 

From the 2018 Fortran standard (https://j3-fortran.org/doc/year/18/18-007r1.pdf), if `stop` is given a constant number, it should return that number as the exit code. `Error stop` should do the same. The difference seems to be only when `stop` or `error stop` are used without arguments.